docs/content/_index.md

```diff
@@ -904,7 +904,7 @@ violation contains server.id if {                   # a server is in the violati
     server.protocols[_] == "telnet"                 # it contains the "telnet" protocol.
 }
 
-public_server contains server if {                  # a server exists in the public_server set if...
+public_servers contains server if {                  # a server exists in the 'public_servers' set if...
```

Comment:
I think you missed one spot where the rule is called (`data.example.public_server[s]`). I think you had that in your original change.

Reply:
You are right, sorry. Will amend the last commit shortly.

Reply:
Thanks! Please squash your commits and sign off when you're done.